### DISCUSSION TITLE
How I can organize durable workers for external API reads?

### DISCUSSION THREAD

**Discussion by oRastor** [brefphp/bref#1587] [General]
Hello, I did not find any information on how I can use bref or serverless to raise a worker that will work constantly (cron that runs every 1 minute is not suitable) and read changes from a third-party API every few seconds and then generate events through the EventBbridge that will be processed by other workers (I understand how it make).

Thanx!

**Comment by mnapoli** [comment]
You can't. AWS Lambda is not made to run a daemon that runs all the time.

All code running on AWS Lambda must be triggered by an event: an HTTP request, a cron, an eventbridge message, etc.

You could look into Fargate for that maybe?

**Comment by oRastor** [comment]
Do I understand correctly, we cannot deploy demons via serverless?

**Reply by mnapoli** [reply]
No, you can't.
